LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015-2016 Slack Technologies, Inc
+Copyright (c) 2015- Slack Technologies, LLC
```

Comment:
I saw a recent message from legal recommending that the copyright year be 2021 in a similar context. Is it OK to keep it open-ended as we have here?

Reply:
Good point. Will check.

Reply:
Confirmed. Just having the original date with the latest entity is appropriate here. No update is needed.